### ISSUE TITLE
Anything special needed for seeding?

### ISSUE THREAD

**Issue by zfLQ2qx2** [varbhat/exatorrent#349]
I downloaded two test torrents - worked flawlessly - but once they completed it looked like all activity dropped off.

The torrents are not stopped, I'm allowing all incoming TCP and UDP traffic to port 42069, outgoing traffic to any address/port, and I even tried setting globalseedratio to 1 in the engine config but I'm not seeing any activity.

Are there any additional steps I need to do?

**Comment by varbhat** [comment]
Nope. No additional steps are required to seed the torrents. I will look into what you have posted. in the meantime, can you also report it at anacrolix/torrent?

**Comment by zfLQ2qx2** [comment]
@varbhat Will do. BTW, thanks for creating this, I had not used bittorrent for several years and when I started looking for clients and I found everything had become closed source and they want people to purchase a monthly subscription. It is great that someone is still doing open source.

**Comment by rihaq** [comment]
Port forward for seeding. Skill issue

**Comment by zfLQ2qx2** [comment]
> Port forward for seeding. Skill issue

@rihaq Chunks were uploading before the torrent completed downloading so I don't think port forwarding is the issue in this case. But no new chunks have uploaded since the torrent was complete.

**Comment by rihaq** [comment]
You've to port forward otherwise seeding willn't work
